### ISSUE TITLE
Slicing time takes more time with larger number of slices

### ISSUE THREAD

**Issue by tuandoan998** [YiguanLiao/YOLOX-SAHI#1]
I use your road.jpg image and perform with 16 number of slices. Processing time is greatly increased compared to 6 slices.
I found the reason is that when using torch.cat(), it is faster to replace it with torch.stack().

But after replacing that, the total time when using batch is still not much faster than standard prediction.

**Comment by DeepLabc** [comment]
Yes, you will find that the time in this part is the preprocessing time including all slices, i.e. the more slices, the longer the processing time.
` batch is still not much faster than standard prediction.`Are you sure using standard SAHI?
